src/js/base/runtime.js

```diff
@@ -265,7 +264,14 @@ function (Namespace, jsnums, codePoint, seedrandom, util) {
 
     var DefaultReprMethods = {
       "string": String,
-      "number": String,
+      //"number": String,
+      "number": function(val) {
+        if (jsnums.isRoughnum(val)) {
+          return val.toString() + '...';
```

Comment:
Does this work on primitive doubles, or does this need to be `String(val) + '...'` instead?